manifest.json

```diff
@@ -1,29 +1,35 @@
 {
-  "name": "SQL Importer",
-  "description": "Import SQL data from your own database",
+  "name": "SQL Connector",
+  "description": "Import data from your SQL databases",
   "picture": "picture.png",
-  "version": "0.2.3",
+  "version": "0.3.0",
   "tags": [
     "incoming",
     "oneColumn"
   ],
   "private_settings": [
     {
+      "type": "string",
+      "format": "title",
+      "name": "connector_info",
+      "title": "Connector Configuration"
+    },
+    {
       "name": "enabled",
-      "title": "Enable Sync",
+      "title": "Enable Sync (default interval is 3 hours)",
       "type": "boolean",
       "default": "false"
     },
     {
       "name": "overwrite",
-      "title": "Overwrite data in Hull in case of a conflict",
+      "title": "Always overwrite data in Hull with values from SQL",
       "type": "boolean",
-      "default": "true"
+      "default": "false"
```

Comment:
All connectors overwrite by default. In that sense it is consistent with what all the other connectors do.

The wording here is threatening because it derives from the import API setting buy in this context, overwrite is expected. 

I think we should stick to `true`.

But what do you think if we reversed the setting to offer to "skip data if there is a value" (a.k.a setIfNull)